package.json

```diff
@@ -43,8 +43,9 @@
     "prompts": "~2.1.0",
     "semver": "^6.1.1",
     "toposort": "^2.0.2",
-    "yargs-parser": "^20.2.4",
-    "uuid": "^8.3.1"
+    "uuid": "^8.3.1",
+    "workspace-tools": "^0.10.2",
```

Comment:
As mentioned in the other comment, do we want to take a dependency on `workspace-tools` as a whole or just `find-yarn-workspace-root`?